pkg/apis/servicecatalog/types.go

```diff
@@ -49,9 +49,9 @@ type BrokerSpec struct {
 	AuthUsername string
 	AuthPassword string
 
-	// CF-specific; move to annotation
-	// CFGUID is the identity of this object for use with the CF SB API.
-	CFGUID string
+	// OSB-specific
+	// OSBGUID is the identity of this object for use with the OSB API.
+	OSBGUID string
```

Comment:
Is there not already a GUID attached to kubernetes objects that can be used for this, or do we feel the need to have a completely new GUID for SB interaction?